### PR TITLE
Missing playwright expect rule

### PR DESCRIPTION
**Pull request by mskelton**
My team has found that it can be easy to forget to await async playwright matchers such as `toHaveSelector`, which can result in false positives since the test needs to wait for the expectation to work properly.  This improves the ESLint plugin to include a new rule that will detect and auto fix this problems.  For example, with this input:

```js
expect(page).toHaveSelector('.form')
```

Auto fixed output:

```js
await expect(page).toHaveSelector('.form')
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,8 +2,18 @@
...
+  "main": "lib/index.js",
   "repository": "https://github.com/mxschmitt/eslint-plugin-jest-playwright",
   "author": "Max Schmitt <max@schmitt.mx>",
-  "license": "MIT"
+  "license": "MIT",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "test": "jest"
```
Comment (mskelton) [inline package.json:13]: Not sure your preference, but might be nice to add a GitHub action to run these tests on builds?

Comment (mxschmitt) [reply package.json:13]: that would be super cool!

**Review by mxschmitt — APPROVED** [review]
that's awesome! Seems to be similar to https://eslint.org/docs/rules/require-await

File: package.json (hunk 1 above)
Comment (mskelton) [reply package.json:13]: Sounds good, I'll set one up

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: Test
+on: [push]
```
Comment (mxschmitt) [inline .github/workflows/test.yml:2]:
```suggestion
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

that should fix it.

**Comment by mskelton** [comment]
@mxschmitt I setup the GH action, and tested it on my fork.  You can see the results here: https://github.com/mskelton/eslint-plugin-jest-playwright/actions/runs/794056637

File: .github/workflows/test.yml (hunk 2 above)
Comment (mxschmitt) [reply .github/workflows/test.yml:2]: That it does not ran inside this Pull Request. They actually allow it, a few days ago it was changed, that the maintainer needs to approve it but generally it's possible.

Yes, mostly such a proposed way is used to also run it when people create pull requests.

Comment (mskelton) [reply .github/workflows/test.yml]: Sweet, thanks!

**Review by mxschmitt — APPROVED** [review]
Awesome work!

**Pull request merged**